modules/ztp-sno-du-configuring-the-operators.adoc

```diff
@@ -4,18 +4,18 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="ztp-sno-du-configuring-the-operators_{context}"]
-= Operator namespaces and Operator groups
+= Operators
 
-{sno-caps} clusters that run DU workloads require the following `OperatorGroup` and `Namespace` custom resources (CRs):
+{sno-caps} clusters that run DU workloads require the following Operators to be installed:
 
-* Local Storage Operator
+* Local Storage Operator or {lvms}
```

Comment:
Only the Local Storage Operator is part of our DU reference. LVMS can optionally be installed if users account for the additional resource costs for it. We need to be clear here though that it is just the Local Storage Operator. Perhaps just leave "Local Storage Operator" and don't include the "or lvms".